### PR TITLE
mwan3: add check_quality for httping

### PR DESCRIPTION
**Pull request by wrsolichin**
Run tested: RUTX12 Router, OpenWrt distribution release 21.02.0

Description: Currently, `check_quality` option is only available for track method `ping`. This PR adds `check_quality` option for track method `httping`.

### CODE REVIEW

**Review by feckert — CHANGES_REQUESTED** [review]
Please bump the PKG_RELEASE number by one

File: net/mwan3/files/usr/sbin/mwan3track [hunk 1]
```diff
@@ -294,14 +294,32 @@ main() {
...
-							WRAP httping -c $count -t $timeout -q "http://$track_ip" &> /dev/null &
+							if [ "$httping_ssl" -eq 1 ]; then
+								WRAP httping -c $count -t $timeout "https://$track_ip" 2> /dev/null > $TRACK_OUTPUT &
+							else
+								WRAP httping -c $count -t $timeout "http://$track_ip" 2> /dev/null > $TRACK_OUTPUT &
+							fi
+							TRACK_PID=$!
+							wait $TRACK_PID
+							ping_status=$?
+							loss="$(sed $TRACK_OUTPUT -ne 's/.* \([0-9]\+\).*% failed.*/\1/p')"
```
[the remark this answers is not in the document]
Comment (wrsolichin) [reply net/mwan3/files/usr/sbin/mwan3track:315]: Updated the commit message to add example output of TRACK_OUTPUT.

**Comment by feckert** [comment]
@wrsolichin 

> > Please bump the PKG_RELEASE number by one
> 
> The current master branch version is 2.11.15, and my current Makefile is 2.11.16. Should I bump it to 2.11.17?

That´s fine :+1:
I will check the new feature and then merge it if everything works as expected.

:+1:

**Comment by wrsolichin** [comment]
> @wrsolichin
> 
> > > Please bump the PKG_RELEASE number by one
> > 
> > 
> > The current master branch version is 2.11.15, and my current Makefile is 2.11.16. Should I bump it to 2.11.17?
> 
> That´s fine 👍 I will check the new feature and then merge it if everything works as expected.
> 
> 👍

Thanks! Let me know if any other changes are needed.

**Comment by feckert** [comment]
Thanks merged!

**Pull request merged**